main.js

```diff
@@ -67,6 +71,13 @@ document.addEventListener('keydown', (event) => {
 
 document.addEventListener('keydown', (event) => {
   if (event.key === ' ') {
-    car1.startCar()
+    if (this.started) {
```

Comment:
The `car` instance should know whether it is started. Right now your global `window` (`this` in your event listener function) knows whether it is started.